examples/nextjs-lexical/app/lexical/editor.tsx

```diff
@@ -87,23 +80,26 @@ export default function Editor() {
 function Threads() {
   const { threads } = useThreads();
 
-  return (
-    <div className="text-sm relative w-[350px] h-full overflow-auto border-l border-border/80">
-      {threads.map((thread) => {
-        return <ThreadWrapper key={thread.id} thread={thread} />;
-      })}
-    </div>
+  const isMobile = useIsMobile();
+
+  return isMobile ? (
+    <FloatingThreads threads={threads} className="w-[350px]" />
+  ) : (
+    <AnchoredThreads threads={threads} className="w-[350px]" />
   );
```

Comment:
@marcbouchenoire Updated to use a `useIsMobile` hook to determine if the user is on mobile depending on screen size. The `useIsMobile` hook uses `useSyncExternalStore` under the hood. Curious if you think this will make understanding this easier or more difficult?

Reply:
I think it's better 👍 I did [move it to its own file](https://github.com/liveblocks/liveblocks/pull/1831/commits/0f9f90ab00612419eb228f5fc23abaf3c0df7f11) given how much an implementation detail it is.